### PR TITLE
New recipe for Corpse-Reviver no.2

### PR DESCRIPTION
**Pull request by Slidd**

### CODE REVIEW

File: src/recipes/corpse-reviver-no2.json [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+        {
+            "quantity": "1",
+            "measure": "dash",
+            "ingredient": "La Fee Parisienne absinthe"
+        }
+    ],
+    "directions": [
+        "Shake all the ingredients with ice and fine strain into a chilled glass."
+    ],
+    "image": "Corpse-Reviver",
```
Comment (alfg) [inline src/recipes/corpse-reviver-no2.json]: Can we match the image filename with the recipe filename?
```suggestion
    "image": "corpse-reviver-no2.jpg",
```

Comment (Slidd) [reply src/recipes/corpse-reviver-no2.json]: Updated.

Comment (alfg) [reply src/recipes/corpse-reviver-no2.json]: Sorry, is it also possible to provide a 600x400px image?

**Comment by Slidd** [comment]
Image updated with correct dimensions

**Comment by alfg** [comment]
Thanks!

**Pull request merged**